### DISCUSSION TITLE
Health Checks for Mirth Connect server behind an Elastic Load Balancer in AWS

### DISCUSSION THREAD

**Discussion by KeithMHughesRoteraAI** [nextgenhealthcare/connect#6054] [Q&A] [answered]
I am wanting to run Mirth Connect in AWS behind an Elastic Load Balancer. I need a health check that the load balancer can hit to make sure the service is running. Is there a URL on the HTTP connector for 8080 that can be used as a health check?

**Comment by pacmano1** [comment, marked as answer]
Just use port 8443, you don't need to use 8080.   Or you can just do you own dumb listener.

And this is a question, not an issue, moving your post to the dicussions area.

**Reply by rinshadka** [reply]
Hi, I am also facing the same issue. I have the service running in 8080 and 8443 ports in an ec2 instance and configured application load balancer for routing the requests in to each services.  But the health checking is not passing.
